bundles-maven-plugin/README.md

```diff
@@ -0,0 +1,89 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+# Apache Metron Bundle Maven Plugin
```

Comment:
I don't disagree with this necessarily.  A couple of thoughts as to why it is the way it is:
1.  we may have archetypes that aren't extensions, so I didn't bury them there
2. at the start @mattf-horton and I talked about having bundles and the plugin actual be external to metron, with the goal being a shared system with NiFi, so I didn't fully integrate them that way.

I am ok with moving them if that is what the consensus is.